e2e-tests/src/test/validators_change.rs

```diff
@@ -16,20 +16,13 @@ pub fn change_validators(config: &Config) -> anyhow::Result<()> {
 
     let connection = RootConnection::new(&config.node, sudo);
 
-    let reserved_before: Vec<AccountId> = connection
-        .as_connection()
-        .get_storage_value("Elections", "NextEraReservedValidators", None)?
-        .unwrap();
+    let reserved_before: Vec<AccountId> =
+        connection.read_storage("Elections", "NextEraReservedValidators");
 
-    let non_reserved_before: Vec<AccountId> = connection
-        .as_connection()
-        .get_storage_value("Elections", "NextEraNonReservedValidators", None)?
-        .unwrap();
+    let non_reserved_before: Vec<AccountId> =
+        connection.read_storage("Elections", "NextEraNonReservedValidators");
```

Comment:
Some other thoughts on naming: we dropped _value_ from a name, but what if we want to add support for double map keys in the future? Unfortunately (in this case; perhaps it's better in the general case) Rust does not support function overload, so we can't use call `connection.read_storage(pallet, key1, key2)`, that would be quite readable.  So I anticipate `read_storage_value` (or `get_storage_value`) is perhaps better, since in the future we can add `read_storage_map` and `read_storage_double_map`. The _value_ in the name is important since it denotes for the reader the return value is a simple value, and this is as well baked into Substrate naming convention that storage keeps `StorageKey`s and values.